html/sos.html

```diff
@@ -0,0 +1,38 @@
+<head>
+  <style>
+    table {
+      font-family: arial, sans-serif;
+      border-collapse: collapse;
+      width: 100%;
+    }
+    td, th {
+      border: 1px solid #dddddd;
+      text-align: left;
+      padding: 8px;
+    }
+  </style>
+  <script>
+  </script>
+</head>
+
+<body>
+  <h1>Current Services (html read in)</h1> 
```

Comment:
Nit: the external file should be the correct version, and the embedded be the fallback. This should not have (html read in) since if this works everything should be fine.